### DISCUSSION TITLE
Can I resize game window to say 2x or 3x only?

### DISCUSSION THREAD

**Discussion by molitar** [Blinue/Magpie#567] [Q&A]
I do not want to do a full screen borderless window I want to just make window size larger without blocking my tool bar.  I can do this with Lossless Scaling but I have not figured out a way to do it in Magpie.  Is it possible to say make it not full screen?

**Comment by luojunyuan** [comment]
[135](https://github-com.translate.goog/Blinue/Magpie/issues/135?_x_tr_sl=zh-CN&_x_tr_tl=en-US) 
Seems like they won't implement it in short term. I wonder does Lossless Scaling mouse cursor works good in window scaling?

**Reply by molitar** [reply]
I have not seen any problems and so far it blows away Magpie as I can just scale window by anything above 1 by 1.1, 1.2 and so on. I scaled an older RPG Maker game by 2.5x instead of using it's own default <F5> and damn it was nice and clean. 

I tested the mouse scaling in Losseless Scaling with this same game with a 2.5x zoom and the mouse cursor seemed to scale very well with the game window.
